superdesk.config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * This is the default configuration file for the Superdesk application. By default,
+ * the app will use the file with the name "superdesk.config.js" found in the current
+ * working directory, but other files may also be specified using relative paths with
+ * the SUPERDESK_CONFIG environment variable or the grunt --config flag.
+ */
+
+module.exports = function(grunt) {
```

Comment:
I agree that a simple JSON object would be better here. You've said during one of our discussions that you'd like this to be a function so we can make use of environment variables. But a function gives too much freedom to do "hacks" to a third party. I will make this into a simple JSON object and name it `superdesk.config.json`.

As for using it live, I think it is better as it is because when we disable a feature, that specific module will not be included into the bundle so when you create a build you will create it containing only the files that are actually used, no extra luggage. This will move us closer to having a core and other pluggable components in my opinion.